input/resources/au-core-server.xml

```diff
@@ -3674,17 +3794,53 @@
 </extension>
 <code value="search-type"/>
 </interaction>
-<referencePolicy value="resolves">
-<extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
-<valueCode value="SHOULD"/>
-</extension>
-</referencePolicy>
-<searchRevInclude value="Provenance:target">
+<interaction>
 <extension url="http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation">
-<valueCode value="SHALL"/>
+<valueCode value="MAY"/>
```

Comment:
Provenance:target is still SHALL. Revert this change.

Reply:
I cannot find Provenance:target as MAY anywhere?!